README.md

```diff
@@ -339,106 +339,348 @@ alpaca.deleteFromWatchlist('xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', "AAPL").then(
 
 ### Data API
 
-#### Get Bars
-
+#### Trades
 ```ts
-getBarsV2(
-  symbol,
-
-  {
-    limit: number,
-    start: date isoformat string yyyy-mm-ddThh:MM:ss-04:00,
-    end: date isoformat string yyyy-mm-ddThh:MM:ss-04:00,
-    timeframe: "1Min" | "1Hour" | "1Day" | "1Week" | "1Month"
-  }
-) => Promise<BarsObject>
+getTradesV2(
+  symbol: string,
+  options: GetTradesParams,
+  config?: any,
+): AsyncGenerator<AlpacaTrade, void, unknown>;
 ```
-###### example
-```js
-let resp = alpaca.getBarsV2(
-    "AAPL",
-    {
-        start: "2021-02-01",
-        end: "2021-02-10",
-        limit: 2,
-        timeframe: "1Day",
-        adjustment: "all",
-    },
-    alpaca.configuration
-);
-const bars = [];
-
-for await (let b of resp) {
-    console.log(b)
-}
+```ts
+getMultiTradesV2(
+  symbols: Array<string>,
+  options: GetTradesParams,
+  config?: any,
+): Promise<Map<string, AlpacaTrade[]>>;
+```
+```ts
+getMultiTradesAsyncV2(
+  symbols: Array<string>,
+  options: GetTradesParams,
+  config?: any,
+): AsyncGenerator<AlpacaTrade, void, unknown>;
 ```
-note: to get the date of response samples you could do this `console.log(new Date(resp['AAPL'][0].startEpochTime*1000))`
-
-#### Latest trade
-
 ```ts
 getLatestTrade(
   symbol: string,
-  config?: any
-) => Promise<AlpacaTrade>
+  config?: any,
+): Promise<AlpacaTrade>;
 ```
-Or if you want to get the latest trades for multiple symbols
 ```ts
 getLatestTrades(
   symbols: Array<string>,
-  config?: any
-): Promise<Map<string, AlpacaTrade>>
+  config?: any,
+): Promise<Map<string, AlpacaTrade>>;
 ```
-###### example
-```js
+##### Example
+```ts
 const trade = await alpaca.getLatestTrade('AAPL');
-
 console.log(trade);
 ```
+###### Output:
+```python
+{
+  Timestamp: '2022-04-14T13:54:24.907840256Z',
+  Exchange: 'P',
+  Price: 169.33,
+  Size: 100,
+  Conditions: [ '@' ],
+  ID: 12272,
+  Tape: 'C'
+}
+```
```

Comment:
Why is this output marked as python ? :thinking:

Reply:
Because it has better colouring on the text. It is the same for the other cases.